examples/benchmark.py

```diff
@@ -10,7 +10,7 @@
 import numpy as np
 
 parser = argparse.ArgumentParser("Running benchmarks on simulator")
-parser.add_argument("--scene", type=str, default="test.glb")
+parser.add_argument("--scene", type=str, required=True)
 parser.add_argument(
     "--max_frames",
     type=int,
```

Comment:
Also, let's add a `--benchmark-semantic` flag so that this script can be run without semantics?